mappings/net/minecraft/block/Material.mapping

```diff
@@ -2,55 +2,99 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
+		COMMENT Material for structure void block.
+	FIELD field_15928 DENSE_ICE Lnet/minecraft/class_3614;
+		COMMENT Material for ice blocks that do not melt.
 	FIELD field_15929 liquid Z
 	FIELD field_15930 EGG Lnet/minecraft/class_3614;
+		COMMENT Material for egg blocks, such as dragon and turtle eggs.
 	FIELD field_15931 WOOL Lnet/minecraft/class_3614;
+		COMMENT Material for wool and bed blocks.
 	FIELD field_15932 WOOD Lnet/minecraft/class_3614;
+		COMMENT Material for wood logs, and things crafted from them. Includes the nether stems (nether wood logs).
 	FIELD field_15933 PISTON Lnet/minecraft/class_3614;
+		COMMENT Material for piston blocks.
 	FIELD field_15934 SNOW_BLOCK Lnet/minecraft/class_3614;
+		COMMENT Material for full sized snow blocks.
 	FIELD field_15935 PLANT Lnet/minecraft/class_3614;
-	FIELD field_15936 CLAY Lnet/minecraft/class_3614;
+		COMMENT Material for plants such as flowers and crops
+	FIELD field_15936 MOB_DERIVED Lnet/minecraft/class_3614;
+		COMMENT Material for blocks that are a product of mob interactions such as honey or slime, or the infested blocks. Includes clay but not bone blocks.
 	FIELD field_15937 CAKE Lnet/minecraft/class_3614;
+		COMMENT Material for cake.
 	FIELD field_15938 BAMBOO_SAPLING Lnet/minecraft/class_3614;
+		COMMENT Material for bamboo saplings.
 	FIELD field_15939 blocksLight Z
 	FIELD field_15940 blocksMovement Z
-	FIELD field_15941 EARTH Lnet/minecraft/class_3614;
+	FIELD field_15941 TOPSOIL Lnet/minecraft/class_3614;
+		COMMENT The top layer of soil. Path, dirt, podzol, soul soil, farmland and similar.
 	FIELD field_15942 GLASS Lnet/minecraft/class_3614;
+		COMMENT Material for glass and glass-like blocks (Includes sea lanterns and conduits).
 	FIELD field_15943 FIRE Lnet/minecraft/class_3614;
+		COMMENT Material for fire blocks.
 	FIELD field_15944 color Lnet/minecraft/class_3620;
-	FIELD field_15945 ORGANIC Lnet/minecraft/class_3614;
+	FIELD field_15945 FODDER Lnet/minecraft/class_3614;
+		COMMENT Food, especially dried hay or feed, for cattle and other livestock. Includes mycelium.
 	FIELD field_15946 BAMBOO Lnet/minecraft/class_3614;
+		COMMENT Material for bamboo.
 	FIELD field_15947 UNDERWATER_PLANT Lnet/minecraft/class_3614;
-	FIELD field_15948 SNOW Lnet/minecraft/class_3614;
-	FIELD field_15949 ANVIL Lnet/minecraft/class_3614;
+		COMMENT Material for underwater plants without the replaceable property.
+	FIELD field_15948 SNOW_LAYER Lnet/minecraft/class_3614;
+		COMMENT Material for non-full blocks of snow. Has the replaceable property.
+	FIELD field_15949 REPAIR_STATION Lnet/minecraft/class_3614;
+		COMMENT Material for blocks that can repair tools, ie. grindstone and anvils.
 	FIELD field_15950 replaceable Z
 	FIELD field_15951 burnable Z
 	FIELD field_15952 BARRIER Lnet/minecraft/class_3614;
+		COMMENT 
 	FIELD field_15953 METAL Lnet/minecraft/class_3614;
-	FIELD field_15954 PUMPKIN Lnet/minecraft/class_3614;
+		COMMENT Material for barrier blocks.
```

Comment:
```suggestion
		COMMENT Material for metal blocks.
```

Reply:
>Pumpkin
>material for --barrier-- metal blocks.

Uuuh....

Reply:
(The comment is for `METAL`)

Reply:
Yes, we should start using split than unified diffs. unified diffs are confusing!

Reply:
![image](https://user-images.githubusercontent.com/7806504/77260817-3a582b00-6c58-11ea-9f92-3957c642f6d0.png)
This is how it actually looks in split diff @sollace